Code/Legacy/CryCommon/IMovieSystem.h

```diff
@@ -1408,40 +1397,6 @@ struct IMovieSystem
     // </interfuscator:shuffle>
 };
 
-inline void SAnimContext::Serialize(XmlNodeRef& xmlNode, bool bLoading)
```

Comment:
was this function not being used? I am not sure if this is used by the "cinematics" part. There are pieces of Cry animation that were not removed to keep some cinematics functionality around

Reply:
Not used on Linux at least, building engine under windows to verify.

Reply:
Windows build checked, and no uses of `SAnimContext::Serialize` encountered :smile: